packages/http-client-js/package.json

```diff
@@ -0,0 +1,79 @@
+{
+  "name": "@typespec/http-client-js",
+  "version": "1.0.0",
+  "type": "module",
+  "private": true,
+  "main": "dist/src/index.js",
+  "scripts": {
+    "build-src": "babel src -d dist/src --extensions .ts,.tsx",
+    "build": "tsc -p . && npm run build-src",
+    "clean": "rimraf ./dist",
+    "lint": "eslint . --max-warnings=0",
+    "lint:fix": "eslint . --fix",
+    "watch-src": "babel src -d dist/src --extensions .ts,.tsx --watch",
+    "watch-tsc": "tsc -p . --watch",
+    "watch": "concurrently --kill-others \"npm run watch-tsc\" \"npm run watch-src\"",
+    "test:scenarios": "vitest run scenarios.test.ts",
+    "test": "npm run test:scenarios",
+    "build-todo": "rimraf sample/output/todo dist/sample/output/todo && tsp compile sample/main.tsp --emit @typespec/http-client-js --output-dir sample/output/todo && babel sample/output/todo -d dist/sample/output/todo --extensions .ts,.tsx && npx prettier sample/output --write",
+    "format": "prettier . --write",
+    "build:test": "tsc -p tsconfig.test.json",
+    "start:server": "npx tsp-spector server start node_modules/@typespec/http-specs/specs --coverageFile ./tsp-spector-coverage-javascript-standard.json --debug",
+    "stop:server": "npx tsp-spector server stop",
+    "test:e2e": "npm run emit:e2e && node eng/scripts/run-e2e-tests.js",
+    "run:e2e": "node eng/scripts/run-e2e-tests.js",
+    "emit:e2e": "node eng/scripts/emit-e2e.js"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "description": "",
+  "peerDependencies": {
+    "@typespec/compiler": "workspace:~",
+    "@typespec/emitter-framework": "workspace:~",
+    "@typespec/http": "workspace:~",
+    "@typespec/http-client": "workspace:~",
+    "@typespec/json-schema": "workspace:~",
+    "@typespec/openapi": "workspace:~",
+    "@typespec/openapi3": "workspace:~",
```

Comment:
why do you need json schema, openapi and openapi3 in the emitter?

Reply:
I used to need them for cadl-ranch tests but not anymore with spector, cleaning up these